src/WebJobs.Extensions.ApiHub/Listener/ApiHubListener.cs

```diff
@@ -134,7 +175,34 @@ public void Dispose()
             }
         }
 
-        private async Task<string> GetLastPollStatusAsync()
+        private async Task MoveToPoisonQueueAsync(ApiHubFile apiHubFile)
+        {
+            CloudQueueClient queueClient = CloudStorageAccount.Parse(this._config.StorageConnectionString).CreateCloudQueueClient();
+
+            var queue = queueClient.GetQueueReference(PoisonQueueName);
+
+            await queue.CreateIfNotExistsAsync();
+
+            var fileStatus = new ApiHubFileInfo
+            {
+                FilePath = apiHubFile.Path,
+                FunctionName = this._functionName,
+                Connection = this._connectionStringSetting
```

Comment:
It's only resolved if he's using [AutoResolve(AllowTokens=false)] on this property. I think ApiHub handles it differently and resolves these on their own.

---
In reply to: [70373036](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/118#discussion_r70373036) [](ancestors = 70373036)